hw/ip/flash_ctrl/data/flash_ctrl.hjson.tpl

```diff
@@ -784,7 +798,94 @@
         { bits: "2",    name: "prog_full",  desc: "Flash program FIFO full"},
         { bits: "3",    name: "prog_empty", desc: "Flash program FIFO empty, software must provide data", resval: "1"},
         { bits: "4",    name: "init_wip",   desc: "Flash controller undergoing init, inclusive of phy init"},
-        { bits: "${8 + bank_width + page_width -1}:8", name: "error_addr", desc: "Flash controller error address."},
+      ]
+    },
+
+    { name: "ERR_CODE",
+      desc: '''
+        Flash error code register.
+        This register tabulates detailed error status of the flash.
+        This is separate from !!OP_STATUS, which is used to indicate the current state of the software initiated
+        flash operation.
+      '''
+      swaccess: "rw",
```

Comment:
should these be `rw1c` accessible such that SW can clear recoverable errors and alerts?

Reply:
yeah i thought about that, but i didn't think these behaved exactly like interrupts. 
i don't really have a strong opinion, i can change if that's what you prefer.